client/package.json

```diff
@@ -5,7 +5,7 @@
         "postinstall": "node ./tasks/install-extensions.js"
     },
     "dependencies": {
-        "superdesk-analytics": "github:superdesk/superdesk-analytics#release/1.33",
+        "superdesk-analytics": "github:superdesk/superdesk-analytics#v1.33.4",
```

Comment:
We should also change server side analytics version to keep it in sync with client one

Reply:
Server is using `develop` branch so we think it's okay to keep it, should we pin it to `v1.33.4` as well?

https://github.com/superdesk/superdesk-belga/blob/0b07b29b816f1693907e409818a706f55225324b/server/requirements.txt#L8

Reply:
IMO it's better to have it in sync, to have the same version on client and on server.
please, try to fix a build.

Reply:
I checked and it seems the build is failed on Node v8 but no problem for Node v10 and v12. Could you check which version of Nodejs travis is using?